man/systemd.resource-control.xml

```diff
@@ -214,6 +214,40 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>CPUSetCpus</varname></term>
+
+        <listitem>
+          <para>Assign specific CPUs to the processes executed. Takes a list of
+            CPU indicies or ranges separated by either whitespace or commas. CPU
+            ranges are specified by the lower and upper CPU indicies separated
+            by dash.
+          </para>
+          <para>Setting the <varname>CPUSetCpus</varname> doesn't guarantee that
+            all of the CPUs will be used by the processes as it may be limitted
+            by parrent units. The effective configuration is reported as
+            <varname>CPUSetCpusEffective</varname>.
+          </para>
```

Comment:
The man page should mention that these directives are supported only with the unified cgroup hierarchy.

Reply:
Fixed, thanks.